content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -214,6 +215,15 @@ To request a larger volume for a PVC, edit the PVC object and specify a larger
 size. This triggers expansion of the volume that backs the underlying `PersistentVolume`. A
 new `PersistentVolume` is never created to satisfy the claim. Instead, an existing volume is resized.
 
+#### CSI Volume expansion
+
+{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+
+CSI volume expansion requires enabling `ExpandCSIVolumes` feature gate and
+running [external resize controller](https://github.com/kubernetes-csi/external-resizer/). You should also look into specific CSI driver documentation and if it
```

Comment:
sorry, I really don't think this is an actionable instruction to a K8s user.  We don't talk about enabling any of the other CSI sidecars, so I don't think we should call it out here.

Let's point to kubernetes-csi documentation on how a driver developer can add resizing support to their driver.  But for an end-user, they should be looking at the CSI driver's documentation on how to enable resizing.

Reply:
okay I have removed the bit about having to run sidecar.